components/script/script_runtime.rs

```diff
@@ -776,3 +781,53 @@ impl Deref for JSContext {
         &self.0
     }
 }
+
+pub struct StreamConsumer(*mut JSStreamConsumer);
```

Comment:
Sorry, one last change that will be necessary - add a `#![allow(dead_code)]` at the top of the file after the license header comment. Otherwise our CI will complain because these new methods and functions are not actually called by any code.

Reply:
Done.